.github/dependabot.yml

```diff
@@ -38,6 +38,7 @@ updates:
   - "jsugarman"
   - "kmahern"
   - "jrmhaig"
+  - "mabley"
```

Comment:
There's just a small typo in the username :) - "mabely"

Reply:
oops, sorry

Reply:
No worries

Reply:
fixed now